joshzarr/FreeRTOS
# Covers all the functions with increased branch and lines

Description
-----------
Coverage is increased to

|             | Hit         | Total       |Coverage     |
| ----------- |-----------  | ----------- | ----------- |
| Lines       | 2832         | 3063         | 92.5 %      |
| Functions   | 214          | 214          | 100.0 %      |
| Branches    | 1512         | 1831         | 82.6 %     |

Test Steps
-----------
Cd into `FreeRTOS/Test/CMock` and then run `make`. Open the generated file is found under `bin/coverage/index.html`

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
